benchmark/README.md

```diff
@@ -0,0 +1,51 @@
+# Benchmark
+
+## Benchmark datasets
+
+The [sqd-collection](https://github.com/optimizers/sqd-collection) repository contains 351 symmetric and quasi-definite linear systems.
+
+```bash
+git clone https://github.com/optimizers/sqd-collection.git
+```
+
+## Running the benchmark
+
+Then, run the benchmark as follows:
+```julia
+using PkgBenchmark
+import LimitedLDLFactorizations
+results = benchmarkpkg("LimitedLDLFactorizations")
+export_markdown("results.md", results)
+```
+
+## Comparing two commits
+
+To compare against the `master` branch
+```julia
+using PkgBenchmark
+import LimitedLDLFactorizations
+judgement = judge("LimitedLDLFactorizations", "master")
+export_markdown("judgement.md", judgement)
```

Comment:
Would be great to also describe how to plot a profile with SolverBenchmark.

Reply:
We don't need at least two solvers for that?

Reply:
oh ok, you want to use LLDL with a different commit? It will be the second solver.

Reply:
Yes.